public/app/plugins/datasource/jaeger/components/AdvancedOptions.tsx

```diff
@@ -1,131 +0,0 @@
-import { css } from '@emotion/css';
-import React, { useState } from 'react';
-import { CSSTransition } from 'react-transition-group';
-
-import { GrafanaTheme } from '@grafana/data';
-import { Icon, InlineField, InlineFieldRow, InlineLabel, Input, useStyles } from '@grafana/ui';
-
-import { JaegerQuery } from '../types';
-
-const durationPlaceholder = 'e.g. 1.2s, 100ms, 500us';
```

Comment:
Nitpick: We probably could have kept this a constant as it is used on 2 separate places.

Reply:
I agree, I will add this constant in `SearchForm.tsx`. I'm debating whether to add it just under the imports or just above the `SearchForm` component. Thoughts?

Reply:
honestly don't think it matters that much probably under imports same way as it was before is fine :)

Reply:
Added in commit https://github.com/grafana/grafana/pull/53774/commits/ccf37b50a6b3acc271c7cbf08ac86acb24868052.